plugins/modules/zos_job_submit.py

```diff
@@ -634,20 +640,25 @@ def run_module():
     module = AnsibleModule(argument_spec=module_args, supports_check_mode=True)
     encoding = module.params.get("encoding")
     if encoding is None:
-        encoding = {"from": DEFAULT_ASCII_CHARSET, "to": DEFAULT_EBCDIC_CHARSET}
+        encoding = {
+            "from": Defaults.DEFAULT_LOCAL_CHARSET,
+            "to": EncodeUtils().remote_charset(),
+        }
     if encoding.get("from") is None:
-        encoding["from"] = DEFAULT_ASCII_CHARSET
+        encoding["from"] = Defaults.DEFAULT_LOCAL_CHARSET
     if encoding.get("to") is None:
-        encoding["to"] = DEFAULT_EBCDIC_CHARSET
+        encoding["to"] = EncodeUtils().remote_charset()
 
     arg_defs = dict(
         src=dict(arg_type="data_set_or_path", required=True),
         wait=dict(arg_type="bool", required=False),
         location=dict(
-            arg_type="str", default="DATA_SET", choices=["DATA_SET", "USS", "LOCAL"],
+            arg_type="str",
+            default="DATA_SET",
+            choices=["DATA_SET", "USS", "LOCAL"],
         ),
-        from_encoding=dict(arg_type="encoding", default=DEFAULT_ASCII_CHARSET),
-        to_encoding=dict(arg_type="encoding", default=DEFAULT_EBCDIC_CHARSET),
+        from_encoding=dict(arg_type="encoding", default=Defaults.DEFAULT_LOCAL_CHARSET),
```

Comment:
Just a quick observation, the `Defaults` class has the following variables:
`DEFAULT_ASCII_CHARSET`, `DEFAULT_EBCDIC_USS_CHARSET` and `DEFAULT_EBCDIC_MVS_CHARSET`.

It should be changed to something like this maybe:
```
from_encoding=dict(arg_type="encoding", default=Defaults.DEFAULT_ASCII_CHARSET),
to_encoding=dict(arg_type="encoding", default=Defaults.DEFAULT_EBCDIC_USS_CHARSET),
```

Reply:
Thanks Asif, for some reason it looks like last merge I performed did not include the latest Defaults, will make the changes!

Reply:
Your comments made much more sense after successfully fetching your updates 😄

Reply:
No worries, happy you were able to get it done! Thanks!